csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -1019,8 +1019,20 @@ void HopperMultipleMatmulScheduler::scheduleEpilogue() {
     // TODO: Use stmatrix to load from registers to shared memory
```

Comment:
We can remove this TODO comment. Also, do we plan to make the stmatrix tile size configurable later? We could have that in MatmulParams.